user_guide_src/source/installation/upgrade_models.rst

```diff
@@ -0,0 +1,71 @@
+Upgrade Models
+###############################################################################
+
+.. contents::
+    :local:
+    :depth: 1
+
+
+Documentations
+============================================================
+
+- `Model Documentation Codeigniter 3.X <http://codeigniter.com/userguide3/general/models.html>`_
+- `Model Documentation Codeigniter 4.X <http://codeigniter.com/user_guide/models/model.html>`_
+
+
+What has been changed
+============================================================
+- The CI4 model has much more functionality, including automatic database connection, basic CRUD, in-model validation, and automatic pagination
+- Since Namespaces has been added in CodeIgniter 4, the models must be changed to support namespaces.
```

Comment:
```suggestion
- The CI4 model has much more functionality, including automatic database connection, basic CRUD, in-model validation, and automatic pagination.
- Since namespaces has been added to CodeIgniter 4, the models must be changed to support namespaces.
```